website/docs/topics/the-anatomy-of-unleash.mdx

```diff
@@ -0,0 +1,195 @@
+
+The end of this guide presents a [short use case, explaining how you might configure Unleash](#use-case) to start working with feature toggles.
+
+
+## The global level
+
+Some things in Unleash are configured and defined on the global level. These options apply across the entire Unleash instance. The most important global configuration options for day-to-day operations are:
+
+- [API access tokens](../reference/api-tokens-and-client-keys.mdx)
+- [Projects](../user_guide/projects.md)
+- [Segments](../reference/segments.mdx)
+- [Strategy types](../user_guide/activation-strategies.md) (including [custom activation strategy types](../advanced/custom-activation-strategy.md))
+- [Tag types](../advanced/tags.md)
+- [Unleash context](../user_guide/unleash-context.md) fields (including [custom context fields](../user_guide/unleash-context.md#custom-context-fields))
+- Users, [user groups](../user_guide/rbac.md#user-groups) and [global roles](../user_guide/rbac.md)
+
+
+## Projects
+
+[**Projects**](../user_guide/projects.md) contain [feature toggles](../reference/feature-toggles.mdx) and their configurations, and a set of active [environments](../user_guide/environments.md).
+
+All Unleash instances must have at least one project at any given time. New instances get a project called “Default”.
+
+Pro and Enterprise customers can create, rename, and delete projects as they wish (as long as there is always **at least one project**). Open-source users, on the other hand, only get access to the default project.
+
+<Figure caption="Unleash projects contain one or more environments." alt="A square labeled 'project' containing another square, labeled 'environment'." img="/img/anatomy-of-unleash-environment.png"/>
+
+## Environments and project environments
+
+<Figure img="/img/anatomy-of-unleash-customer-tiers.png" caption="Feature toggles can be activated or deactivated independently in different environments. For instance, a feature toggle can be active in the development environment, and deactivated in the production environment. Even if their configuration is otherwise identical, deactivated feature toggles will never be considered enabled."/>
+
+[**Environments**](../user_guide/environments.md) in Unleash let you change how a feature toggle works in your application’s different environments. For instance, while you are developing a feature, it’s likely that you’ll want it to be available in the development environment, but not in the production environment: environments let you do that. You might also want to enable a feature for only some users in a production environment, but for no users in a development environment: environments let you do that.
```

Comment:
Kinda want to use "your" here to point out that this is referring to your servers and not Unleash config but back to what makes your eyes happy

```suggestion
[**Environments**](../user_guide/environments.md) in Unleash let you change how a feature toggle works in your application’s different environments. For instance, while you are developing a feature, it’s likely that you’ll want it to be available in your development environment, but not in your production environment: environments let you do that. You might also want to enable a feature for only some users in your production environment, but for no users in your development environment: environments let you do that.
```